routes/sports.js

```diff
@@ -1,39 +1,37 @@
 var express = require('express');
 var router = express.Router();
-const { checkBody } = require('../modules/checkBody');
+const { checkBody } = require('../utils/checkBody');
 
-const Sport = require('../models/sports');
+const SportService = require ('../services/Sport');
+
+router.post('/', async (req, res) => {
 
-router.post('/', (req, res) => {
-    // Check if all fiels are filled out
     if (!checkBody(req.body, ['label'])) {
-        res.json({ result: false, error: 'Missing or empty fields' });
+        res.json({ result: false, error: 'Missing or empty fields.' });
         return;
-    }
-
-    const sportToSave = req.body.label.toLowerCase()
-
-    Sport.findOne({ label: sportToSave }).then(data => {
-        if (data === null) {
-            const newSport = new Sport({
-                label: sportToSave
-            });
-            newSport.save().then(newDoc => {
-                res.json({ result: true, token: newDoc.token });
-            });
-        }
-        else {
-            // User already exists in database
-            res.json({ result: false, error: 'Sport already exists' });
-        }
-    })
+    };
+
+    const { label } = req.body;
+    const sportName = label[0].toUpperCase() + label.slice(1).toLowerCase();
```

Comment:
This could be directly formatted in the frontend ?

Reply:
no, as this controller is only used by developers to create new sports